lib/scan.ts

```diff
@@ -153,8 +153,36 @@ export function getExcludedPatterns(
 
   const config: DotSnykConfig = dotSnyk.parse(policyFilePath);
 
-  return [
-    policyFilePath,
-    ...dotSnyk.toAbsolutePaths(projectRoot, config?.exclude?.global),
-  ];
+  const today = new Date();
+
+  const paths: readonly string[] | undefined = config?.exclude?.global
+    ?.filter((item) => {
+      if (typeof item === 'object') {
+        const key = Object.keys(item)[0];
```

Comment:
Something to consider, maybe the block inside the if should be extracted to it's own function. Since it's a bigger one.

It could be extracted to a function called `hasExpired` or similar, and placed in the dotSnyk-package.